### ISSUE TITLE
[FEATURE] Test Set for LibPQ Connection

### ISSUE THREAD

**Issue by TheCedarPrince** [JuliaDatabases/DBConnector.jl#1]
Currently, the test set for PostgreSQL support via LibPQ.jl does not exist yet. We need a test set that covers the following functions within `postgresql.jl`:

- [ ] `DBInterface.connect` - this is based off of the documentation for the [DBInterface `connect` function](https://juliadatabases.org/DBInterface.jl/dev/#DBInterface.connect)
- [ ] `DBInterface.prepare` - this is based off of the documentation for the [DBInterface `prepare` function](https://juliadatabases.org/DBInterface.jl/dev/#DBInterface.prepare)
- [ ] `DBInterface.execute` - this is based off of the documentation for the [DBInterface `execute` function](https://juliadatabases.org/DBInterface.jl/dev/#DBInterface.execute)
- [ ] `_dbconnect` - this function is the core dispatch within DBConnector.jl and is exported for a user

To create tests initially, a PostgreSQL database can be used either locally or on a server. Most likely for those interested in addressing this problem, one will have to set-up a local PostgreSQL instance for testing. 

Furthermore, it will generally be difficult to create tests individually for each of these functions so we could perhaps instead adopt an integration test approach where we have a test set that relies on each of these three functions working together.

**Comment by TheCedarPrince** [comment]
This will become a lot easier thanks to the upcoming PR that will be added to LibPQ.jl: https://github.com/iamed2/LibPQ.jl/pull/271 !

**Comment by TheCedarPrince** [comment]
Until this PR is finished, we may have to use this fork until this functionality is merged into LibPQ itself: `https://github.com/chris-b1/LibPQ.jl#dbinterface`. But, that should unblock the complications in adding this feature.
